ebssurrogate/files/migrations/db/migrations/20221013153608_test.sql

```diff
@@ -0,0 +1,5 @@
+-- migrate:up
+
+
+-- migrate:down
+
```

Comment:
can be deleted?

Reply:
ah, I didn't realise, let me go ahead and delete it.